src/state/user/reducer.ts

```diff
@@ -5,9 +5,15 @@ import { ConnectionType } from '../../connection/types'
 import { SupportedLocale } from '../../constants/locales'
 import { DEFAULT_DEADLINE_FROM_NOW } from '../../constants/misc'
 import { RouterPreference } from '../../state/routing/types'
+import {
+  deletePersistedOriginCountry,
+  getPersistedOriginCountry,
+  setPersistedOriginCountry,
+} from './originCountryStorage'
 import { SerializedPair, SerializedToken, SlippageTolerance } from './types'
 
 const selectedWallet = getPersistedConnectionMeta()?.type
+const originCountry = getPersistedOriginCountry()
```

Comment:
I'd like to see a follow up where we group these localStorage-duplicated data into their own folder, with some shared infra between them so we're not re-writing it every time we need something accessible on load.

Reply:
by follow up, you mean follow up pr right? or do you expect this in this pr?

Reply:
Follow up PR, yeah